go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/golang/snappy v1.0.0
 	github.com/google/gopacket v1.1.19
 	github.com/gorilla/mux v1.8.1
-	github.com/grafana/dskit v0.0.0-20250407073127-5ff25a4351b6
+	github.com/grafana/dskit v0.0.0-20250417074529-d8712729db54
```

Comment:
can we use the sha from main? `15fe9b42d47ddac4d4c947def386494a5a39cc60`

Reply:
The main revision unfortunately pulls in 7b78bfe441ed988fe6369d6d2efd089d334aaa9b as well, which makes porting to backend-enterprise much more difficult, as at least Transmog also needs updating.